### PR TITLE
update default scan timeout to 3 minutes

### PR DESCRIPTION
**Pull request by pankhurisaxena28**

### CODE REVIEW

File: sarif.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (sethvargo) [inline sarif.json:1]: Did you mean to include this file @pankhurisaxena28? It seems unused in this PR.

Comment (pankhurisaxena28) [reply sarif.json:1]: This was generated automatically as a result of `npm build` and `npm run test`.

Comment (pankhurisaxena28) [reply sarif.json]: Nonetheless, we do not want this file to be present in the repository. Therefore, I have deleted the file.

**Review by verbanicm — DISMISSED** [review]

**Review by kishorekgarg — APPROVED** [review]

**Review by kishorekgarg — APPROVED** [review]

**Review by verbanicm — APPROVED** [review]

File: sarif.json (hunk 1 above)
Comment (sethvargo) [reply sarif.json]: We may want to add it to a `.gitignore` then in the future.

**Pull request merged**